### PR TITLE
Wikis / REST API v2 changes

### PR DESCRIPTION
**Pull request by snazy**

### CODE REVIEW

File: api/model/src/main/java/org/projectnessie/model/Documentation.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  @jakarta.validation.constraints.NotNull
+  @Value.Parameter(order = 1)
+  String getMimeType();
+
+  /** URI of the documentation. If present, {@link #getText()} must not be specified. */
+  @Nullable
+  @jakarta.annotation.Nullable
+  @Value.Parameter(order = 1)
+  @JsonInclude(JsonInclude.Include.NON_NULL)
+  String getLocation();
```
[the remark this answers is not in the document]
Comment (snazy) [reply api/model/src/main/java/org/projectnessie/model/Documentation.java:45]: Makes sense, updated `Documentation`

File: api/model/src/main/java/org/projectnessie/model/Documentation.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+  /** Documentation text. If present, {@link #getLocation()} must not be specified. */
+  @Nullable
+  @jakarta.annotation.Nullable
+  @Value.Parameter(order = 1)
+  @JsonInclude(JsonInclude.Include.NON_NULL)
+  String getText();
```
[the remark this answers is not in the document]
Comment (snazy) [reply api/model/src/main/java/org/projectnessie/model/Documentation.java:52]: Nope. Images and other potentially embedded resources are out of scope.

File: api/model/src/main/java/org/projectnessie/model/GetMultipleContentsResponse.java [hunk 3]
```diff
@@ -66,14 +67,27 @@ interface ContentWithKey {
...
     @jakarta.validation.constraints.NotNull
+    @Value.Parameter(order = 2)
     Content getContent();
 
+    @Nullable
+    @jakarta.annotation.Nullable
+    @JsonView(Views.V2.class)
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    @Value.Parameter(order = 3)
+    Documentation getDocumentation();
```
[the remark this answers is not in the document]
Comment (snazy) [reply api/model/src/main/java/org/projectnessie/model/GetMultipleContentsResponse.java:83]: Yes - exactly one documentation, which can be some markdown or some pointer (URI), according to the mime-type. Images et al are our of scope and need to be managed externally.

[the remark this answers is not in the document]
Comment (dimas-b) [reply api/model/src/main/java/org/projectnessie/model/GetMultipleContentsResponse.java:83]: SGTM

**Review by dimas-b — APPROVED** [review]

**Pull request merged**